pyproject.toml

```diff
@@ -34,11 +34,19 @@ addopts = "-v --mypy -p no:warnings --cov=myproject --cov-report=html --doctest-
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
+[tool.ruff]
+target-version = "py311"
```

Comment:
Does this value need to be kept in-line with the python version specified earlier in this file?

Reply:
Kind of. It's used for ruff's pyupgrade rules, so it will upgrade your Python syntax to 3.11 features in this case.